docs/guide/catalog/catalog_api/routing_system.md

```diff
@@ -1,6 +1,6 @@
-# Routing system [[% include 'snippets/commerce_badge.md' %]]
+# Routing system
 
-[[= product_name_com =]] uses its own routing/URL matching solution to avoid conflicting with the [[= product_name =]] routing system
+[[= product_name =]] uses its own routing/URL matching solution to avoid conflicting with the [[= product_name =]] routing system
```

Comment:
```suggestion
[[= product_name =]] uses its own routing/URL matching solution to avoid conflicting with the [[= product_name =]] routing system
```

Now we have "DXP uses a different solution to avoid conflict with DXP" - should it be "the Commerce bundle", "the shop"?